### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,37 @@
+const button = document.querySelector("button");
+const counter = document.querySelector("select");
+const cardsList = document.getElementById("cards-list");
 
+for (let i = 1; i <= 10; i += 1) {
+  const option = document.createElement("option");
+  option.text = i;
+  option.value = i;
+  counter.appendChild(option);
+}
```
Comment (mikeboyle) [inline index.js:12]: We would also accept people manually adding the `<option>` elements to the html directly. No need to change this though

**Review by mikeboyle — COMMENTED** [review]
So far so good! There may be more things depending on changes to the tests (or adding a bonus challenge to the lab)

File: index.js [hunk 2]
```diff
@@ -1 +1,37 @@
...
+  async function drawCards() {
+    const count = counter.value;
+    const drawCardsResponse = await axios(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=${count}`);
+    const { cards } = drawCardsResponse.data;
+
+    cardsList.innerHTML = "";
+
+    for (const card of cards) {
+      const cardImage = document.createElement("img");
+      cardImage.className = "card";
```
Comment (mikeboyle) [inline index.js:37]: Left this comment on the tests as well. The instructions say to use the class name `image`. I'm fine with whatever as long as the tests, instructions, and solution use the same class name.

File: index.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply index.js:12]: I can add a comment :)

**Review by mikeboyle — COMMENTED** [review]
Changes look good but can you chase down why one of the tests is still failing?

**Comment by JoshuaKGoldberg** [comment]
Ah, merge shenanigans.

**Comment by mikeboyle** [comment]
Shenanigans! 🍀

**Comment by JoshuaKGoldberg** [comment]
Wait, hang on, this isn't merge shenanigans -- it just passes locally & doesn't pass in CI. Ugh.

**Comment by JoshuaKGoldberg** [comment]
@mikeboyle any interest in https://github.com/archfz/cypress-terminal-report? I'm having a hard time debugging without it.

**Comment by mikeboyle** [comment]
@JoshuaKGoldberg feel free to add that library, great call.

**Comment by JoshuaKGoldberg** [comment]
Super. I'm going to do some debugging on this PR; you might want to unsubscribe 😄. When it's all done I'll merge everything together & probably end up sending a PR to the template.

**Comment by JoshuaKGoldberg** [comment]
Hmm, looks like the fixture isn't sending applying:

```
          cy:xhr ➟  GET https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1 (59 ms)
                    Status: 200 - OK
```

https://app.circleci.com/pipelines/github/joinpursuit/Pursuit-Core-Web-Async-Await-Lab/9/workflows/df5248da-ea50-4eec-9127-c5041d8064b5/jobs/9/steps

**Comment by mikeboyle** [comment]
@JoshuaKGoldberg Is this line maybe not doing anything or not doing what we expect it to do?
```
cy.wait("@newDeck");
```

**Comment by JoshuaKGoldberg** [comment]
For fear of being arrogant _and_ wrong, I ... think it's doing what I want? But I don't feel confident in that. Playing around...

**Comment by JoshuaKGoldberg** [comment]
Yeah, I ended up not needing the `.wait`ing on specific intercepts. 🤷

**Comment by mikeboyle** [comment]
So `.wait` made it ... not wait? 
Glad it's resolved!

**Review by mikeboyle — COMMENTED** [review]
🙏 Thank you for sorting out that confounding test issue!
~Two~ One very small nit:
- can you remove the `Hello World` from the solution?
~- is the `N cards remaining` text covered up by the cards?~ (never mind, this is the bonus)
- When those two things are resolved, can you reupload the gif to the lab README?

**Comment by JoshuaKGoldberg** [comment]
Voila:
Screen recording of drawing 5 cards, then 3, then 1

**Comment by JoshuaKGoldberg** [comment]
...and we're back to the CI issues! Bizarre. I'm heading out for now but will take a look later today.

**Comment by mikeboyle** [comment]
Closing to hide solution

**Pull request closed without merging**